modules/metering-reports.adoc

```diff
@@ -0,0 +1,332 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-about-reports.adoc
+[id="metering-reports_{context}"]
+= Reports
+
+The `Report` custom resource is used to manage the execution and status of reports.
+Metering produces reports derived from usage data sources which can be used in further analysis and filtering.
+
+A single `Report` resource represents a job manages a database table and updates it with new information according to a schedule, and exposes the data in that table via the reporting-operator HTTP API.
+Reports with a `spec.schedule` field set are always running, and will track what time periods it has collected data for, ensuring that if Metering is shutdown or unavailable for an extended period of time, it will backfill the data starting where it left off.
+If the schedule is unset, then the Report will run once for the time specified by the `reportingStart` and `reportingEnd`.
+By default, reports will wait for `ReportDataSources` to have the progressed in their import process to cover the report period being processed.
+If the report has a schedule, it will wait until the period currently being processed has been covered by the import process.
```

Comment:
We need to ensure that the usage of Report is consistent. If it's an API object, our latest style guidelines say it should be capitalized.

Reply:
Ah, I thought Kevin said only the first occurence does, at least in a sentence.

Reply:
Should I do this across the board?

Reply:
I think so. @lamek Is there updated guidance for this? I was out on leave for a few months and could be wrong.

Reply:
No updated usage. What you say is correct. Let's capitalize Report throughout. We're doing the same for the word Operator and Pod.

Reply:
Sounds good. I'll make that change.